src/BloomExe/Spreadsheet/SpreadsheetIO.cs

```diff
@@ -237,7 +236,7 @@ int embedImage(string imageSrcPath, int rowNum, int colNum)
 				catch (Exception ex)
 				{
 					progress?.Message("Spreadsheet.ExportFailed", "",
-						"Export failed: " + ex.Message,
+						"Export failed: " + ex.ToString(),
```

Comment:
I'm not sure we want the full exception in this case. Isn't this shown to the user?

Also, even though you didn't do this, the l10n of this is wrong. We add the exception message to a localized string without a placeholder. We either need a placeholder, or we need to append the exception after localizing. (I haven't looked in the .xlf to see what is there currently.)